Sources/NIOHTTP2/HTTP2StreamChannel.swift

```diff
@@ -728,7 +728,7 @@ private extension HTTP2StreamChannel {
     /// Drop all pending reads.
     private func dropPendingReads() {
         /// We don't need to report the dropped reads, just remove them all.
-        self.pendingReads.removeAll()
+        self.pendingReads.removeAll(keepingCapacity: true)
```

Comment:
Should we worry about this potentially keeping an unneeded/too-big capacity for the duration of the stream?

Reply:
No, this is only called when the channel is closing.